Chapters/1-4-Chapter4.md

```diff
@@ -0,0 +1,232 @@
+
+The train slowed to a stop and opened its doors, but again only a few people boarded. The roll had not sustained me, and I was really craving a real meal. _Perhaps I ought to treat myself to something nice in the capital before I get banished to the wilderness... No... I don't have time for that._
+
+I took out the letter to review it and think about how I might deliver it. The letter was marred with creases from all the travel, but luckily the fibers and the ink held up. I saw the dots and the circle again, but now I could make out the sentences.
+
+```
+Obscura Observation
+Melna,
+
+I am following up on our conversations about colder temperatures in Heiko and other western sectors. After 20 Tova of observation of Ara, we have measured no measured reduction in stellar activity and no change in stellar size. However, we did detect a decrease in incident light. This is likely caused by a large diffuse obstruction made up of smaller objects. For brevity, we will call this collection "the obscura". It is difficult to estimate the size of each object, but they seem to span the entire view we have of Ara. Upon closer observation, we conclude that they are unlikely to be caused by a natural phenomenon as the obstructions seem to be growing in number and are moving in tandem so to be evenly-spaced.
+
+If left alone, we calculate that the obscura will cause colder temperatures, weaker winds, and decreased precipitation, which will lead to most seasonal crops to become dormant in six days. In ten days, the temperatures and incident light will be low enough to freeze and kill all known crops. We advise immediate action to clear the obstruction. We will continue to monitor and update you everyday, and would appreciate your advice on how we can communicate more efficiently in this time.
+
+- Jathi
+```
+
+"What is this??" I murmured out loud, then glanced over at the old man to make sure he was still sleeping. I could feel my skin getting wet and cold in anxiety. _Kipnu... Melna... what will happen to them? Our home. The garden... And how will the senate receive this? They would dismiss me as a lunatic if I interrupt them for something this outlandish. If I get sent to Ulukoma, while they prepare for the cold, what will I come back to?_ It was too much to think of. I closed my eyes, and I rode the waves of anxiety back down. _I must deliver this message. I can think about the future later. For now, I'm doing the right thing for all of Palinan and that's all I need._ I folded up the letter and put it back in my bag, returning my thoughts to my head and my gaze toward the vast darkness of Ulu.
+
+"You look like you've seen something you'd rather not have," said the man, startling me.
+
+I shot my eyes toward his, gray, washed-out.
+
+"Pardon?" I jabbed. I was at the edge of my day's limit for politeness.
+
+He chuckled, "Kipnu told me you were just like Melna. You're far worse."
```

Comment:
How did he know about Kipnu?

Reply:
Yea I don't think this connection is important. I was going to make it more so as we go on, but nah..